src/store/config.js

```diff
@@ -10,10 +10,10 @@ import * as reducers from './reducers';
 
 const middlewares = [thunk, createActionBuffer(REHYDRATE)];
 
-// if (__DEV__) {
-//   const { createLogger } = require('redux-logger');
-//   middlewares.push(createLogger({}));
-// }
+if (__DEV__) {
+  const { createLogger } = require('redux-logger');
+  middlewares.push(createLogger({}));
+}
```

Comment:
@edencakir try https://github.com/jhen0409/react-native-debugger I found it more productive than `redux-logger`.